### PR TITLE
travis CI

### PR DESCRIPTION
**Pull request by rpitonak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        - 'sudo apt-get install go-md2man'
+env:
+        - VERSION=x.y OS=fedora
+        - VERSION=x.y OS=centos
+        - VERSION=x.y OS=rhel
+        - VERSION=x.z OS=fedora
+        - VERSION=x.z OS=centos
+        - VERSION=x.z OS=rhel
+script:
+        - make build VERSION=$VERSION TARGET=$OS
```
Comment (bkabrda) [inline .travis.yml:18]: This should probably also include `make test ...`, right?

Comment (TomasTomecek) [reply .travis.yml:18]: Yes. As far as I know, we don't have any tests here, right?

**Review by TomasTomecek — APPROVED** [review]
LGTM, we need to figure out the tests; I checked the test runs and all passed

File: .travis.yml (hunk 1 above)
Comment (bkabrda) [reply .travis.yml:18]: Right... so we should probably do some followup work to include and run example tests.

**Comment by TomasTomecek** [comment]
Let's merge and continue working on tests here: https://github.com/container-images/container-image-template/pull/32

**Pull request merged**